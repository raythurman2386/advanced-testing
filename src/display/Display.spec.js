import React from 'react';
import * as rtl from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Display from './Display';

afterEach(rtl.cleanup);

describe('Display Suite', () => {
  it('Snapshot Test', () => {
    // ready to test!
    const wrapper = rtl.render(<Display />);
    expect(wrapper.asFragment()).toMatchSnapshot();
  });
});
