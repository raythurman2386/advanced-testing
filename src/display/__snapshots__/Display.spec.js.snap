// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Suite Snapshot Test 1`] = `
<DocumentFragment>
  <div
    class="display panel"
  >
    <div
      class="led green-led"
    >
      Unlocked
    </div>
    <div
      class="led green-led"
    >
      Open
    </div>
  </div>
</DocumentFragment>
`;
